storybook/src/docs/components/DataGrid/DataGrid.stories.tsx

```diff
@@ -574,6 +543,25 @@ export const _CrudMoreActionsMenu = {
                         />
                     </ToolbarItem>
                 </DataGridToolbar>
+            );
+        }
+
+        return (
+            <Box sx={{ height: 600, width: "100%" }}>
```

Comment:
This should be enough 🙂 

```suggestion
            <Box height={600}>
```